File: examples/museum/oseg_generated/csharp/ListSpecialEventsDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class ListSpecialEventsDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var response = new EventsApi(config).ListSpecialEvents(
                startDate: DateOnly.Parse("2023-02-23"),
                endDate: DateOnly.Parse("2023-04-18"),
                page: 2,
                limit: 15
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#ListSpecialEvents: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/museum/oseg_generated/ruby/ListSpecialEventsDefaultExample.rb
require "openapimuseum_client"

OpenApiMuseumClient.configure do |config|
end

begin
    response = OpenApiMuseumClient::EventsApi.new.list_special_events(
        {
            start_date: Date.parse("2023-02-23").to_date,
            end_date: Date.parse("2023-04-18").to_date,
            page: 2,
            limit: 15,
        },
    )

    p response
rescue OpenApiMuseumClient::ApiError => e
    puts "Exception when calling Events#list_special_events: #{e}"
end
